### ISSUE TITLE
Lovelace: Entity Button Card forces title to be all caps

### ISSUE THREAD

**Issue by Lopton** [home-assistant/home-assistant-polymer#2407]
**Home Assistant release with the issue:**
0.84.6 

**Last working Home Assistant release (if known):**
N/A

**Browser and Operating System:**
All browsers / O/S

**Description of problem:**
The Entity Button Card changes the titles/names of the entities to be all caps. This seems to be the only component in HA that forces the titles/names to all caps. Every other component displays the titles / names as the user types them.

**Comment by Lopton** [comment]
Since this was new to me, I went and read the Material Design Standards on buttons (https://material.io/design/components/buttons.html). So I'm curious why the Home Assistant project has decided of all the button standards that the uppercase was the one to follow, but ignores the other portions. 

Of note in every example provided in the document, the text of the button is a different color, indicating the button has a different action. But in Home Assistant the text is not different, instead it looks identical to all the other text, except it is capital. But if I were to label my entities in the all caps it would look them appear identical to the buttons. 

Here is a picture of my current dashboard, note how the buttons look very similar to the Motion Detector indicator? Only it looks like I typed the buttons in all caps. This seems to violate the intent / spirit of the Material Design guidelines. 

Not only that, but the home assistant "buttons" actually perform the exact same actions as the not buttons, so in the end the user experiences the same actions from the buttons, and not buttons, further blurring the lines between the two, making the reasoning for all caps only confusing to my second user (the wife)...

My two cents.
